### PR TITLE
[SL-UP] Update WiseConnect to v3.3.3

### PR DESCRIPTION
**Pull request by arun-silabs**
This PR contains the changes required for updating WiseConnect to 3.3.3.

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -223,7 +223,7 @@
 [submodule "third_party/silabs/wifi_sdk"]
 	path = third_party/silabs/wifi_sdk
 	url = https://github.com/SiliconLabs/wiseconnect.git
-	branch = v3.3.2
```
Comment (mykrupp) [inline .gitmodules:226]: This isn't necessary to do in matter_sdk, if we want to do this update it should be done in CSA. We use the wifi_sdk from the matter_extension, so we can still comeplete the task to move to 3.3.3 by updating matter_extensions wifi_sdk submodule.

File: third_party/silabs/matter_support
Comment (mykrupp) [inline third_party/silabs/matter_support:1]: Same comment on this, these should only be updated in CSA.

File: third_party/silabs/wifi_sdk
Comment (mykrupp) [inline third_party/silabs/wifi_sdk:1]: Same comment on this, these should only be updated in CSA.

**Review by mkardous-silabs — CHANGES_REQUESTED** [review]
@arun-silabs We can still cherry-pick changes to th  matter v1.4 branch. Instead of a cherry-pick PR, please open a PR to port these changes to CSA v1.4 branch once the PR towards master is merged.

**Pull request closed without merging**